lib/Doctrine/CouchDB/CouchDBClient.php

```diff
@@ -143,15 +143,39 @@ public function getUuids($count = 1)
     }
 
     /**
-     * Find a document by ID and return the HTTP response.
+     * Find a document by ID and return the HTTP response. If
+     * $getAllLeafRevisions is true, documents for all leaf revisions are
+     * retrieved and $revisions parameter is ignored. If $getAllLeafRevisions
+     * is false and $revisions is not null, only the specified leaf revisions
+     * are retrieved.
      *
-     * @param  string $id
+     * @param string $id
+     * @param bool $getAllLeafRevisions
```

Comment:
Maybe we can name this parameter to something shorter that is more aligned with the CouchDB API? Maybe `$allOpenRevs` ?